src/Event.php

```diff
@@ -33,8 +34,23 @@ public function acknowledge(Message $message)
         array_map(function ($name) use ($message) {
             $this->emitter->emit($name, $message);
         }, [
-            static::QUEUE_ACKNOWLEDGE,
-            sprintf('%s.%s', static::QUEUE_ACKNOWLEDGE, $message->handler())
+            static::MESSAGE_ACKNOWLEDGE,
+            sprintf('%s.%s', static::MESSAGE_ACKNOWLEDGE, $message->handler())
```

Comment:
Only reason I'd held off on moving that to its own method is I think @ameech plans to do that with thephpleague/event once a related issue with event arguments is resolved.